src/include/options.h

```diff
@@ -970,6 +970,12 @@ struct uae_prefs
 	int input_contact_bounce;
 	int input_device_match_mask;
 
+	bool vkbd_hires;
```

Comment:
Since this is also Amiberry-specific, perhaps it would make sense to move the options under the `#ifdef AMIBERRY` section below? That way, it's easier to spot when doing future merges from WinUAE, for example.